src/caches/low_storage_rk_caches.jl

```diff
@@ -0,0 +1,366 @@
+
+@cache struct CarpenterKennedy2N54Cache{uType,rateType,TabType} <: OrdinaryDiffEqMutableCache
```

Comment:
Just make one low-storage 2N cache and have the different algorithms feed a new tableau to it.

Reply:
That's a good idea, I will do that.